config/core/deployments/pingsource-mt-adapter.yaml

```diff
@@ -85,3 +85,23 @@ spec:
               cpu: 1000m
               memory: 2048Mi
       serviceAccountName: pingsource-mt-adapter
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    eventing.knative.dev/release: devel
+    app.kubernetes.io/name: pingsource-mt-adapter
+    app.kubernetes.io/version: devel
+    app.kubernetes.io/part-of: knative-eventing
```

Comment:
These are standard K8s [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels). Is this something followed across the code base @matzew ?

Reply:
It is being rolled out 

https://github.com/knative/community/issues/667

eventing, serving, operator manifests are done. The deployments/services created by Eventing CRDs(Broker,etc) need to be updated to include these labels as well.